runhouse/logger.py

```diff
@@ -20,6 +21,11 @@ class ColoredFormatter:
     def get_color(cls, color: str):
         return cls.COLORS.get(color, "")
 
+    @classmethod
+    def format_log(cls, text):
```

Comment:
can we make a TODO or some note here that this is more of a temp solution for the time being until we update the whole logging architecture